app/controllers/attachments_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class AttachmentsController < ApplicationController
```

Comment:
Should the name of this controller be `FileAttachmentController` as we're using FileAttachment for the model.

Reply:
This one is a bit tied up in "attachments" being the public naming of the concern, but FileAttachment being the only type of attachment and a private concern.

Since we link to insert attachment I believe the URL should be `/attachments` and it feels more natural for the controller to follow that name.

Reply:
I think we were coming from the idea that we're using the concept 'file attachment' to make room for 'html attachment' when it comes along. Maybe we're planning to use this controller for HTML attachments as well, but if not, would it be reasonable to change the models to follow the public naming instead?